zoo-css-challenge/README.md

```diff
@@ -38,30 +34,26 @@ Change the background colour to #483C46, the section titles to #F4743B, and the
 
 Make the first paragraph have a larger font size.
 
-Again, the purple headings are hard to read on dark backgrounds, change them to a lighter colour.
-
 ## 5) News
 
 Can the title of this section be centred in the middle of this page?
 
 Add another news item with the title with today's date, and the title of ‘Which big cat are you?’
 
-## 6) Membership
+## 6) Learning
 
-Give the paragraph a maximum width of 700 pixels, and keep in the middle by adjusting the margin.
+The buttons are a little hard to read. We’d like the buttons to have larger text, and also to change colour when you hover and focus on them.
 
-## 7) Learning
+We don’t like the hard corners here. Can you give the container in the section some rounded corners?
 
-The buttons are a little hard to read. We’d like the buttons to have larger text, and also to change colour when you hover over them.
+## 7) Membership
 
-We don’t like the hard corners here. Can you give the container in the section some rounded corners?
+Give the paragraph a maximum width of 700 pixels, and keep in the middle by adjusting the margin.
 
 ## 8) Programs
 
 The cards are stacked on top of each other which looks like a poor use of space. Change these so that they display horizontally across the screen.
 
-Change each of the ‘Find out more’ links to a button, each one should have a different colour.
```

Comment:
It sounds like this task is to change the `a` to a `button`, which shouldn't be done if the element links to another document.